spyderlib/widgets/sourcecode/base.py

```diff
@@ -1093,10 +1093,16 @@ def wheelEvent(self, event):
         # This feature is disabled on MacOS, see Issue 1510
         if sys.platform != 'darwin':
             if event.modifiers() & Qt.ControlModifier:
-                if event.delta() < 0:
-                    self.zoom_out.emit()
-                elif event.delta() > 0:
-                    self.zoom_in.emit()
+                if hasattr(event, 'angleDelta'):
+                    if event.angleDelta().y() < 0:
+                        self.zoom_out.emit()
+                    elif event.angleDelta().y() > 0:
+                        self.zoom_in.emit()
+                elif hasattr(event, 'delta'):
+                    if event.delta() < 0:
+                        self.zoom_out.emit()
+                    elif event.delta() > 0:
+                        self.zoom_in.emit()

```

Comment:
It's another bug related to PyQt5.5 . I fixed it and created a new pull request. I don't know why it is shown here...